athena_cfn/README.md

```diff
@@ -0,0 +1,193 @@
+Unfortunately, at this time `Source=Custom` Roles are **not** compatible with accessing
+Athena.  Therefore, if you have not already, from the Quilt Admin Settings you
+must first:
+
+1. Create an empty `Source=Quilt` Role, e.g. "AthenaQuiltAccess"
+1. Create a new `Source=Quilt` Policy with access (usually "Read-write") to relevant S3 Buckets, and attach it to that Role
+1. Assign that Role to the Quilt Users that need to access Athena
+
+See the [Users and Roles](https://docs.quiltdata.com/catalog/admin)
+documentation for more details about using Admin Settings to create and assign Roles and Policies.
+
+## II. Upload Athena Configuration files to S3
+
+There are two Athena templates in this directory,
+which must be uploaded directly to S3
+(as they are too large to read directly into CloudFormation)
+
+*  `athena-cfn.yml` is only run once, and creates a results bucket and
+workgroup usable by the entire account (with appropriate permissions).
+
+* `athena-bucket-cfn.yml` must be run *once for each Quilt bucket*
+that Athena should query. This creates bucket-specific tables and views which
+wrap the underlying Quilt metadata.
+
+To install these in S3:
+
+1. Go to Amazon S3 -> Buckets  in the AWS Console for the region where most
+of your buckets are, e.g.:
+https://s3.console.aws.amazon.com/s3/buckets?region=us-east-1
+1. Choose any bucket you can read and write to
+1. Create or navigate to an appropriate folder
+1. Select Upload -> Add files
+1. Select both `athena-cfn.yml` and `athena-bucket-cfn.yml` from this repository
+1. Click Upload
+1. Command-click on each filename to open it in a new browser window
+1. Remember or copy and paste the `https` 'Object URL' for each file
+
+## III. Create Athena Workgroup
+
+### A. Install `athena-cfn.yml` CloudFormation template
+
+Note: If you store large amounts of data in multiple regions, you may need to
+create and manage one such stack per _region_ (rather than per _account_).
+
+1. Go to CloudFormation -> Stacks in the AWS Console for the region where most
+of your buckets are, e.g.:
+https://us-east-1.console.aws.amazon.com/cloudformation/home?region=us-east-1#/stacks
+1. Click dropdown "Create stack" -> "With new resources (standard)"
+1. Select "Template is ready" (default)
+1. Select "Amazon S3 URL" (default)
+1. Paste in the `athena-cfn.yml` Object URL from above
+1. Click Next
+1. Enter "Stack name", e.g.: `athena-quilt`
+1. Click Next
+1. Click Next
+1. Review then check "I acknowledge that AWS CloudFormation might create IAM resources"
+1. Click "Create change set"
+1. Wait until "Status = CREATE_COMPLETE" (may need to refresh using upper-right "cycle" icon)
+1. Review Changes
+1. Click "Execute" in upper right
+1. Select "Roll back all stack resources" (default)
+1. Click "Execute change set"
+
+You need to wait until the `athena-cfn.yml` stack creation completes,
+so you can use its Outputs for the following steps.
+
+### B. Extend Quilt's Permission Boundary
```

Comment:
Please link to main docs on this. Also need a separate PR that modifies the Athena page on docs.quiltdata.com and points here (if you haven't already).

Reply:
https://github.com/quiltdata/quilt/pull/3110